app/models/student_loans/school_eligibility.rb

```diff
@@ -1,5 +1,6 @@
 module StudentLoans
   class SchoolEligibility
+    START_DATE = Date.new(2018, 4, 6)
```

Comment:
Could this const have a more explicit name? POLICY_START_DATE, just to be super clear what it represents?